packages/backend-defaults/src/entrypoints/rootLogger/index.ts

```diff
@@ -14,5 +14,8 @@
  * limitations under the License.
  */
 
-export { rootLoggerServiceFactory } from './rootLoggerServiceFactory';
+export {
+  createRootLogger,
```

Comment:
For some reason running `yarn build` on this package does _not_ include the `createRootLogger` function in the generated `.d.ts` file. However, the export does appear in the `.cjs.js` file. So the runtime export is there, but the type definition is missing, making this function inaccessible outside the package.

I have tried multiple combinations of things to get this working without success. Any help here would be greatly appreciated!